### PR TITLE
Minecodes.azure version 1.1.1

### PR DESCRIPTION
**Pull request by Minecodes**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/78763)

### CODE REVIEW

File: manifests/m/Minecodes/azure/1.1.1/Minecodes.azure.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Minecodes.azure
```
Comment (Trenly) [inline manifests/m/Minecodes/azure/1.1.1/Minecodes.azure.installer.yaml:4]:
```suggestion
PackageIdentifier: Minecodes.4zur3
```
This is best to match `Publisher.PackageName`

File: manifests/m/Minecodes/azure/1.1.1/Minecodes.azure.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Minecodes.azure
```
Comment (Trenly) [inline manifests/m/Minecodes/azure/1.1.1/Minecodes.azure.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Minecodes.4zur3
```
This is best to match `Publisher.PackageName`

File: manifests/m/Minecodes/azure/1.1.1/Minecodes.azure.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Minecodes.azure
```
Comment (Trenly) [inline manifests/m/Minecodes/azure/1.1.1/Minecodes.azure.yaml:4]:
```suggestion
PackageIdentifier: Minecodes.4zur3
```
This is best to match `Publisher.PackageName`

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/78763#pullrequestreview-1099230440

File: manifests/m/Minecodes/4zur3/1.1.1/Minecodes.4zur3.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Minecodes.4zur3
+PackageVersion: 1.1.1
```
Comment (ItzLevvie) [inline manifests/m/Minecodes/4zur3/1.1.1/Minecodes.4zur3.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.1.1.0
```
This must match Control Panel.

You'll also need to change `manifests/m/Minecodes/4zur3/1.1.1` to `manifests/m/Minecodes/4zur3/1.1.1.0`

File: manifests/m/Minecodes/4zur3/1.1.1/Minecodes.4zur3.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Minecodes.4zur3
+PackageVersion: 1.1.1
```
Comment (ItzLevvie) [inline manifests/m/Minecodes/4zur3/1.1.1/Minecodes.4zur3.yaml:5]:
```suggestion
PackageVersion: 1.1.1.0
```
This must match Control Panel.

You'll also need to change `manifests/m/Minecodes/4zur3/1.1.1` to `manifests/m/Minecodes/4zur3/1.1.1.0`

File: manifests/m/Minecodes/4zur3/1.1.1/Minecodes.4zur3.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Minecodes.4zur3
+PackageVersion: 1.1.1
```
Comment (ItzLevvie) [inline manifests/m/Minecodes/4zur3/1.1.1/Minecodes.4zur3.installer.yaml:5]:
```suggestion
PackageVersion: 1.1.1.0
```
This must match Control Panel.

You'll also need to change `manifests/m/Minecodes/4zur3/1.1.1` to `manifests/m/Minecodes/4zur3/1.1.1.0`

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**